src/api/ledger/ledger.js

```diff
@@ -0,0 +1,26 @@
+/* @flow */
+'use strict';
+const utils = require('./utils');
+const validate = utils.common.validate;
+const composeAsync = utils.common.composeAsync;
+const parseLedger = require('./parse/ledger');
+
+function getLedgerAsync(options, callback) {
+  validate.getLedgerOptions(options);
+
+  const request = {
+    ledger: options.ledgerVersion || 'validated',
+    expand: options.includeAllData,
+    transactions: options.includeTransactions,
+    accounts: options.includeAccounts
+  };
+
+  this.remote.requestLedger(request,
+    composeAsync(response => parseLedger(response.ledger), callback));
+}
+
+function getLedger(options={}) {
+  return utils.promisify(getLedgerAsync.bind(this))(options);
```

Comment:
IMO performance shouldn't ever be ignored, at least obvious stuff like this. The performance difference here is fairly significant, and it adds up when you've got a whole stack of functions that call Function.bind (as the case seems to be here), and are called many times throughout the lifetime of a program.  I also find the `call` version slightly more readable as you also mentioned, so it seems like a no brainer